source/tutorials/write-update-documents.txt

```diff
@@ -101,42 +94,43 @@ To learn more about update operators, see
 Update One Document
 -------------------
 
-To update a single document from a MongoDB collection, call the ``updateOne()``
+To update a single document in a MongoDB collection, call the ``updateOne()``
 method and pass your query filter as the first argument. Then, pass your
-update operators as the second argument to the ``updateOne()`` method. After you call the
-``updateOne()`` method, chain the ``subscribe()`` method and pass a subscriber
-as an argument.
+update operators as the second argument to the ``updateOne()`` method. Finally,
+pass the ``updateOne()`` method to the static ``Mono.from()`` method from the
+``Mono Publisher``. In {+java-rs+}, you must use ``Publisher`` implementations to control how data
+is transmitted in your application. The ``Mono Publisher`` is a ``Publisher`` implementation from the Project Reactor library. To learn more
+about ``Mono``, see `Mono <https://projectreactor.io/docs/core/release/reference/#mono>`__ in the Project
+Reactor documentation.
 
 The following example uses ``Updates.set()`` to update the ``name``
 value of a matching document from ``"Bagels N Buns"`` to ``"2 Bagels 2 Buns"``:
 
 .. code-block:: java
    :copyable: true
 
-    ObservableSubscriber<UpdateResult> updateOneSubscriber = new SubscriberHelpers.OperationSubscriber<>();
-    restaurants.updateOne(eq("name", "Bagels N Buns"), set("name", "2
-    Bagels 2 Buns")).subscribe(updateOneSubscriber);
-    updateOneSubscriber.await();
+    Mono.from(restaurants.updateOne(eq("name", "Bagels N Buns"), set("name", "2 Bagels 2 Buns"))).block();
```

Comment:
This comment applies to the sentence just above this code examples (Git won't let me comment there):

I think it's a bit confusing to say "The following example uses Updates.set()" when the set call isn't structured in that way. It might be better to say that it's using the updateOne method:

"The following example uses the ``updateOne()`` method to..."